src/future.ml

```diff
@@ -411,11 +498,13 @@ module Scheduler = struct
     match jobs with
     | [] -> ()
     | first :: others ->
-      Format.eprintf "\nWaiting for the following jobs to finish: %t@."
-        (fun ppf ->
-           Format.fprintf ppf "[@{<id>%d@}]" first.id;
-           List.iter others ~f:(fun job ->
-             Format.fprintf ppf ", [@{<id>%d@}]" job.id));
+      if !Clflags.verbose then begin
```

Comment:
I think we should still print something here when `!verbose = false`. If a compilation command takes long after an error is reported, we end up wondering what's happening. We could just print the number of jobs we are waiting for